app/src/main/java/com/quran/labs/androidquran/dao/audio/AudioPlaybackSettings.kt

```diff
@@ -0,0 +1,12 @@
+package com.quran.labs.androidquran.dao.audio
+
+import android.os.Parcelable
+import com.quran.data.model.SuraAyah
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class AudioPlaybackSettings(val start: SuraAyah,
+                                 val end: SuraAyah,
+                                 val verseRepeatCount: Int = 0,
+                                 val rangeRepeatCount: Int = 0,
+                                 val enforceRange: Boolean = false) : Parcelable
```

Comment:
what if, instead of doing this, we remove this class (and the corresponding changes to read/write this) - and instead do the work in the `AudioService`?